packages/node_modules/@webex/internal-plugin-ediscovery/src/transforms.js

```diff
@@ -339,9 +367,10 @@ class Transforms {
         space.spaceName = decryptedSpaceName;
       })
       .catch((reason) => {
-        ctx.webex.logger.error(`Decrypt space name error for space ${space.spaceId}: ${reason}`);
-        // add error property to space info - this error will be recorded in the downloader
-        space.error = reason;
+        ctx.webex.logger.warn(`Decrypt space name error for space ${space.spaceId}: ${reason}`);
+        // this can fail if we attempt to decrypt an unencrypted spacename, so we report a warning rather than an error
+        // add warn property to space info - this warning will be recorded in the downloader
+        space.warn = reason;
```

Comment:
If we pass an unencrypted string here it manifests as a SyntaxError: Unexpected token I in JSON at position 0. We could potentially interpret this as a warning and anything else as an error but I'm not sure if that's guaranteed to be the only error we receive.

Reply:
AFAIK decryption never fails. Instead it'll rolls/shifts meaningless encrypted bytes to more meaningless bytes.

But the code does an auth check. This essentially compares a hash to a hash we stored. If they don't match it throws an error.

That means this code is checking if the hashes match and when they don't threats it as a warning. I think it needs to be an error.